homeassistant/components/backup/websocket.py

```diff
@@ -351,7 +357,15 @@ async def handle_config_info(
                 vol.Optional("days"): vol.Any(int, None),
             },
         ),
-        vol.Optional("schedule"): vol.All(str, vol.Coerce(ScheduleState)),
+        vol.Optional("schedule"): vol.Schema(
+            {
+                vol.Optional("recurrence"): vol.All(str, vol.Coerce(ScheduleState)),
+                vol.Optional("time"): vol.All(
+                    vol.Schema({"hour": vol.Range(0, 23), "minute": vol.Range(0, 59)}),
+                    time_from_dict,
+                ),
```

Comment:
I was looking around the code base a bit, but not sure how we usually send times. Instead of a dict, maybe a string "hh:mm" would be better?